src/server/api/routes/router.js

```diff
@@ -22,7 +26,12 @@ router.use("/mail", mailRouter);
 
 router.use("/token", refreshTokenRouter);
 
+router.use("/vessels", vesselRouter);
+
 // /api/organizations
 router.use("/organizations", organizationRouter);
 
+// /api/organizations/organization_id/vessels
+router.use("/organizations/:organization_id/vessels", vesselRouter);
```

Comment:
You actually need to put this endpoint in the `organization.router.js`. I realize I should probably have broken things up different and made this endpoint part of the organization task.

Reply:
I have kept same as before